### ISSUE TITLE
The compute node fails to be created in two different LANs.

### ISSUE THREAD

**Issue by leebobby** [openpbs/openpbs#2231]
The network diagram is：
![image](https://user-images.githubusercontent.com/16457850/106561127-1c8b7000-6563-11eb-8e95-c2fa0d0681b5.png)

In mom_log there is a error:
![image](https://user-images.githubusercontent.com/16457850/106561305-670cec80-6563-11eb-9f0a-9433fa54772c.png)
But two node can ping with each other.
![image](https://user-images.githubusercontent.com/16457850/106561447-a6d3d400-6563-11eb-9022-d254d107395f.png)
![image](https://user-images.githubusercontent.com/16457850/106561466-afc4a580-6563-11eb-8227-1a7e3e75d182.png)

In pbs_server the vm186's state is : 
![image](https://user-images.githubusercontent.com/16457850/106561485-bbb06780-6563-11eb-8d58-0c87a6db1b2a.png)

I have check the firewall is closed, I want to know what might be wrong.

**Comment by drtoss** [comment]
Could you show the contents of /etc/pbs.conf on vm186 and on node26?

Also, what does the pbs_comm log say, if anything, when you start pbs on vm186? The log should be in /var/spool/pbs/comm_logs/ on the pbs_server host (node26).

**Comment by leebobby** [comment]
> Could you show the contents of /etc/pbs.conf on vm186 and on node26?
> 
> Also, what does the pbs_comm log say, if anything, when you start pbs on vm186? The log should be in /var/spool/pbs/comm_logs/ on the pbs_server host (node26).

Thanks for your answer!
/etc/pbs.conf on vm186
![image](https://user-images.githubusercontent.com/16457850/106832425-9e4add00-66cc-11eb-96a4-67455e0d28c8.png)
/etc/pbs.conf on node26
![image](https://user-images.githubusercontent.com/16457850/106832392-912dee00-66cc-11eb-8c3e-0b7673662f65.png)

The comm_log is：
![image](https://user-images.githubusercontent.com/16457850/106832717-2630e700-66cd-11eb-844e-623f0c6e12cd.png)

It seems the IP address problem.
8.0.2.168 is Node26's LAN IP, Not global IP. Node26 has used NAT for IP translation, but vm186 only global IP

**Comment by drtoss** [comment]
Yes, this looks like a NAT problem. That is, NAT is happening when it should not.

I need a better picture of the physical network layout. From the logs, it looks as if vm186 is using node26 to provide NAT services. Is that correct? Could you show the routing tables on node26 and on vm186?

`ip route show
`

Also, could you run traceroutes each way between node26 and vm186?
